engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/UniverseWrapper.java

```diff
@@ -13,14 +14,15 @@ public class UniverseWrapper {
     private  String seed;
     private  boolean loadingAsServer;
     private  String gameName;
-    private WorldSetupWrapper targetWorld;
+    private WorldConfigurator worldConfigurator;
+    private WorldGenerator worldGenerator;
 
     public void setSeed(String seed) {
         this.seed = seed;
     }
 
     public String getSeed() {
-        return seed;
+        return seed == null ? "" : seed;
```

Comment:
Is the `null` seed important for anything? If not, we could just initialize it to the empty string as default (and assert that `setSeed` does not accept `null`.